backend/README.md

```diff
@@ -32,7 +32,7 @@ export TOURNEYNIZER_URL_PROD=jdbc:postgresql://localhost:5432/tourneynizer-prod
 export TOURNEYNIZER_USERNAME_PROD=[The username you set up for the db]
 export TOURNEYNIZER_PASSWORD_PROD=[The password you set up for the db]
 ```
-Then, run `source variables.env` to load the environment variables. Currently only the environment variables ending in `DEV` need to be set to run, and the ones ending in `TEST` to run tests.
+Then, run `source .env` to load the environment variables. Currently only the environment variables ending in `DEV` need to be set to run, and the ones ending in `TEST` to run tests.
```

Comment:
Why did we change this?

Reply:
I messed up when initially writing the readme. The .gitignore only has .env in it so variables.env won't be hidden.